var/spack/repos/builtin/packages/py-bx-python/package.py

```diff
@@ -11,10 +11,18 @@ class PyBxPython(PythonPackage):
     to allow for rapid implementation of genome scale analyses."""
 
     homepage = "https://github.com/bxlab/bx-python"
-    url      = "https://github.com/bxlab/bx-python/archive/v0.7.4.tar.gz"
+    url      = "https://pypi.io/packages/source/b/bx-python/bx-python-0.8.8.tar.gz"
 
-    version('0.7.4', sha256='1066d1e56d062d0661f23c19942eb757bd7ab7cb8bc7d89a72fdc3931c995cb4')
+    version('0.8.8', sha256='ad0808ab19c007e8beebadc31827e0d7560ac0e935f1100fb8cc93607400bb47')
+    version('0.7.4',
+            sha256='1066d1e56d062d0661f23c19942eb757bd7ab7cb8bc7d89a72fdc3931c995cb4',
+            url="https://github.com/bxlab/bx-python/archive/v0.7.4.tar.gz")
 
+    depends_on('python@2.4:2.7', type=('build', 'run'), when='@:0.7')
```

Comment:
Are you sure it didn't support Python 3? Didn't it still have a `py-six` dependency back then? I would go with:
```suggestion
    depends_on('python@2.4:', type=('build', 'run'), when='@:0.7')
```
just to be on the safe side.

Reply:
No, it does not build with python-3.

The `py-six` dependency is not listed in the `setup.py` file for version 0.7.4. I left it unconstrained because it was originally there, presumably for a reason. Should I constrain that to '@8:'?

Reply:
Ah yes, in that case please restrict the six dependency to `@8:` and disregard the above comment.